bokeh/charts/builder.py

```diff
@@ -216,6 +216,10 @@ class Builder(HasProps):
         is (Column, Ascending).
         """)
 
+    source = Instance(ColumnDataSource)
+
+    hover = Either(List(Tuple(String, String)), List(String), Bool, default=None)
```

Comment:
Ok, I'll think about it. Initial thought is it is broader than the kwarg tooltip input to hover, but i like tooltips as the name either way.